README.md

```diff
@@ -145,16 +145,36 @@ There is only a limited number of such operations and they are only implemented
 ### Time Graph Tooltip
 Currently, the **Time Graph Tooltip** is populated when selecting a state in a Time Graph view.
 
+## New contributors
+⚠️ **Linux only!** The development environment for this project only works on Linux.
+
+### Explore the project context
+* **Check out the [tracevizlabs](https://github.com/tuxology/tracevizlab).** You don't need to complete all of them, but reading a couple is good context. Lab cycles 0 and 1 are especially useful.
+* **Browse recent conference presentations.** They give an overview of the trace viewer's goals, architecture, and functionality.
+  * _Using Theia to take trace analysis and visualization to the next level_, [slides](https://www.eclipsecon.org/sites/default/files/slides/EclipseConEurope2019-TraceCompass-Theia.pdf) - Bernd Hufmann, Ericsson AB (EclipseCon 2019)
```

Comment:
Since you added the video below, you could add the corresponding video here as well: https://www.youtube.com/watch?v=Fysg1mOadik.